core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -168,6 +168,10 @@ public static void recordGauge(String name, long count) {
         gauge(classQualifier, name, count);
     }
 
+    public static void recordCounter(String name, long count, String... additionalTags) {
```

Comment:
Do you plan to use this method?

Reply:
Yes, i noticed that we have a public method for gauge but not for a counter. Further, I plan to use the counter.